checker/build.gradle

```diff
@@ -1085,6 +1093,39 @@ task wpiPlumeLibTest(group: 'Verification') {
     }
 }
 
+// Unzip JSpecify conformance tests using the artifact
+task unzipConformanceTestSuite(group: 'Verification') {
+    description 'Unzips the conformance test suite into the build directory.'
+    copy {
+        from zipTree(configurations.conformanceTestSuite.singleFile)
+        into "${buildDir}/conformance-test-suite"
+    }
+}
+
+// Run JSpecify conformance tests
+task conformanceTests(type: Test, group: 'Verification') {
+    description 'Runs the conformance tests.'
+    include '**/*ConformanceTest.class'
```

Comment:
Make this the full name, to make it easier to find and consistent with the `exclude` above.